pkg/kubelet/dockershim/BUILD

```diff
@@ -53,9 +53,9 @@ go_library(
         "//pkg/security/apparmor:go_default_library",
         "//pkg/util/exec:go_default_library",
         "//pkg/util/hash:go_default_library",
-        "//pkg/util/term:go_default_library",
         "//vendor/github.com/blang/semver:go_default_library",
         "//vendor/github.com/docker/docker/pkg/jsonmessage:go_default_library",
+        "//vendor/github.com/docker/docker/pkg/term:go_default_library",
```

Comment:
I guess my question is, is `//vendor/github.com/docker/docker/pkg/term` very different from `//pkg/util/term` ? Can we replace `//pkg/util/term` entirely with `//vendor/github.com/docker/docker/pkg/term`?